tests/NSubstitute.Analyzers.Tests.CSharp/CodeFixProviderTests/ReceivedInReceivedInOrderCodeFixProviderTests/ReceivedAsExtensionMethodTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace NSubstitute.Analyzers.Tests.CSharp.CodeFixProviderTests.ReceivedInReceivedInOrderCodeFixProviderTests;
+
+public class ReceivedAsExtensionMethodTests : ReceivedInReceivedInOrderCodeFixVerifier
+{
+    [Fact]
+    public override async Task RemovesReceivedChecks_WhenReceivedChecksHasNoArguments()
+    {
+        var oldSource = @"using NSubstitute;
+using NSubstitute.ReceivedExtensions;
+
+namespace MyNamespace
+{
+    public interface IFoo
+    {
+        int Bar();
+        int Bar(int x);
+    }
+
+    public class FooTests
+    {
+        public void Test()
+        {
+            var substitute = NSubstitute.Substitute.For<IFoo>();
+            Received.InOrder(() =>
+            { 
+                substitute.Received().Bar();
+                substitute.Received().Bar(Arg.Any<int>());
```

Comment:
**suggestion:** maybe add a `substitute.ReceivedWithAnyArgs().Bar(Arg.Any<int>()))` call to this test as well?

Reply:
added